src/components/material/MaterialDescription.tsx

```diff
@@ -42,11 +42,11 @@ const MaterialDescription: React.FC<MaterialDescriptionProps> = ({ work }) => {
       )}
       <div className="material-description__links mt-32">
         {inSeries &&
-          inSeries.map((seriesItem, i) => {
-            return (
+          inSeries.map((seriesItem, i) =>
+            seriesItem.numberInSeries?.number ? (
```

Comment:
I think the same is happening in the [search result item](https://github.com/reload/dpl-react/blob/730d8ac11dcb95a86bb625cd8f36f93e7a873add/src/components/search-result-list/search-result-list-item/search-result-list-item.tsx#L122-L135)?
 I think you should put an effort in making a component fro showing this part, which can be used in both places.